### DISCUSSION TITLE
[FEAT] Align text on left

### DISCUSSION THREAD

**Discussion by JonathanBuchh** [vslinko/obsidian-outliner#113] [Ideas]
I love this plugin and I have a small feature request. When toggling `Improve the style of your lists`, first line get misaligned from the rest. It is slightly to the right of other lines. I would imagine this could be improved with CSS.
<img width="776" alt="Screen Shot 2021-04-30 at 7 25 52 AM" src="https://user-images.githubusercontent.com/63243849/116688870-4fd47400-a985-11eb-97ba-e934eb8b8d2e.png">

**Comment by vslinko** [comment]
@JonathanBuchh Thank you for your idea!

I'm moving the issue to the other ideas.
